app/controllers/GreeterRouter.scala

```diff
@@ -18,7 +18,7 @@ import scala.concurrent.Future
   * Is there a nice place where we can ask the user to 'binder.bind(classOf[GreeterService]).to(classOf[GreeterServiceImpl])'?
   */
 @Singleton
-class GreeterRouter @Inject()(cc: ControllerComponents, impl: GreeterServiceImpl)(implicit mat: Materializer)
+class GreeterRouter @Inject()(cc: ControllerComponents, impl: GreeterServiceImpl, client: GreeterServiceClient)(implicit mat: Materializer)
```

Comment:
are you injecting the client in the router just to proof that your guice module is correct?

Reply:
correct, this was just to check that I could